.github/workflows/checks.yml

```diff
@@ -202,14 +202,12 @@ jobs:
     strategy:
       fail-fast: false
     env:
-      GOERLI_INTEGRATION_RPC_URL: ${{ secrets.INTEGRATION_RPC_URL }}
-      GOERLI_TESTNET_RPC_URL: ${{ secrets.TESTNET_RPC_URL }}
       SEPOLIA_INTEGRATION_RPC_URL: ${{ secrets.SEPOLIA_INTEGRATION_RPC_URL }}
       SEPOLIA_TESTNET_RPC_URL: ${{ secrets.SEPOLIA_TESTNET_RPC_URL }}
-      GOERLI_INTEGRATION_ACCOUNT_PRIVATE_KEY: ${{ secrets.INTEGRATION_ACCOUNT_PRIVATE_KEY }}
-      GOERLI_INTEGRATION_ACCOUNT_ADDRESS: ${{ secrets.INTEGRATION_ACCOUNT_ADDRESS }}
-      GOERLI_TESTNET_ACCOUNT_PRIVATE_KEY: ${{ secrets.TESTNET_ACCOUNT_PRIVATE_KEY }}
-      GOERLI_TESTNET_ACCOUNT_ADDRESS: ${{ secrets.TESTNET_ACCOUNT_ADDRESS }}
+      SEPOLIA_TESTNET_ACCOUNT_ADDRESS: ${{ secrets.SEPOLIA_TESTNET_ACCOUNT_ADDRESS }}
+      SEPOLIA_TESTNET_ACCOUNT_PRIVATE_KEY: ${{ secrets.SEPOLIA_TESTNET_ACCOUNT_PRIVATE_KEY }}
+      SEPOLIA_INTEGRATION_ACCOUNT_PRIVATE_KEY: ${{ secrets.SEPOLIA_INTEGRATION_ACCOUNT_PRIVATE_KEY }}
+      SEPOLIA_INTEGRATION_ACCOUNT_ADDRESS: ${{ secrets.SEPOLIA_INTEGRATION_ACCOUNT_ADDRESS }}
```

Comment:
This accounts were previously created by us, right?

Reply:
Yes, it's taken from a pinned post in our slack